ros2nodl/ros2nodl/verb/show.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the
+# GNU Limited General Public License version 3, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Limited General Public License for more details.
+#
+# You should have received a copy of the GNU Limited General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+
+import sys
+from typing import TYPE_CHECKING
+
+import nodl
+from nodl.errors import DuplicateNodeError, NoNoDLFilesError
+from ros2nodl.api import (
+    NoDLFileNameCompleter,
+    show_nodl_parsed,
+    show_nodl_raw,
+)
+from ros2nodl.verb import VerbExtension
+from ros2pkg.api import package_name_completer, PackageNotFoundError
+
+if TYPE_CHECKING:  # pragma: no cover
+    import argparse
+
+
+class ShowVerb(VerbExtension):
+    """Show NoDL data."""
+
+    def add_arguments(self, parser: 'argparse.ArgumentParser', *_):
+        parser.add_argument(  # type: ignore
+            'package_name', help='Name of the package to show.'
+        ).completer = package_name_completer
+
+        command_group = parser.add_mutually_exclusive_group(required=True)
+        command_group.add_argument(
+            '-a',
+            '--all',
+            '--all-files',
+            default=False,
+            action='store_true',
+            help='Combine all .nodl.xml files and display the combined output.',
+        )
+        command_group.add_argument(  # type: ignore
+            '-f', '--file', nargs='+', help='Specific .nodl.xml file(s) to display.'
+        ).completer = NoDLFileNameCompleter()
```

Comment:
I would have expected this to be executable name rather than a file path. This route has some assumed knowledge (that nodl files exist, where they're installed, etc.). What is the reason we're doing it this way?

Reply:
Fixed in 62602dd